crates/codegen/runtime/cargo/crate/src/extensions/mod.rs

```diff
@@ -6,3 +6,5 @@ pub mod compilation;
 
 #[cfg(feature = "__experimental_bindings_api")]
 pub mod bindings;
+
+pub mod semver;
```

Comment:
nit suggestion: I suggest using the same name as the owning "public" module. i.e. `parser` or `utils`...